lib/create-metadata-object.js

```diff
@@ -0,0 +1,6 @@
+  module.exports = (data) => ({
+        id: data.id,
+        govtrack_id: data.govtrack_id || null,
+        displayName: data.displayName,
+        in_office: !!data.in_office,
```

Comment:
Is this the not not? Why not just use `data.in_office`?

Reply:
it's incase it's undefined or doesn't exist. it's a way of converting a truthy thing to a boolean